packages/openapi-v2/README.md

```diff
@@ -0,0 +1,77 @@
+@loopback/openapi-v2
+
+This package contains:
+
+- Decorators that describe LoopBack artifacts as metadata.
```

Comment:
Decorators that apply REST api mapping metadata to controller classes and their members.

Reply:
@raymondfeng as I have mentioned in https://github.com/strongloop/loopback-next/pull/804#issuecomment-354792283, my intension to have this package is to organize all the openapi related decorators and spec generation functions together. Not only for controller classes....

Reply:
@raymondfeng How about "Decorators that describe LoopBack artifacts as openapi metadata."?  I missed an important word "openapi" in this commit.

Reply:
These decorators are created for application developers to describe REST mapping of controller classes, right?